tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/PackageCheckType.cs

```diff
@@ -28,5 +28,10 @@ public enum PackageCheckType
     /// <summary>
     /// Run cspell.
     /// </summary>
-    Cspell
+    Cspell,
+    
+    /// <summary>
+    /// Fix changelog issues using microagent tools.
+    /// </summary>
+    ChangelogFix
```

Comment:
note for self -- we dont want this to be called as a part of run-checks all right?

Reply:
need to fix that to make sure